packages/ember-routing-htmlbars/tests/helpers/element_action_test.js

```diff
@@ -1002,6 +1085,29 @@ QUnit.test('should respect preventDefault=false option if provided', function()
   equal(event.isDefaultPrevented(), false, 'should not preventDefault');
 });
 
+QUnit.test('should respect preventDefault=false option if provided bound', function() {
+  view = EmberView.create({
```

Comment:
I think I agreed on meeting @rwjblue after 2.3 was released to plan moving tests away from `Ember.View`. While new tests suites are easy to start with `Ember.Component`, most test files using views have already their `setup` and `teardown` based on that.

I rather write this with `Ember.View` and start working on a plan to migrate tests one complete file at a time.